lite2/client.go

```diff
@@ -427,7 +427,7 @@ func (c *Client) Stop() {
 //  - header has not been verified yet
```

Comment:
//  - header expired, therefore can't be trusted (ErrOldHeaderExpired);

needs to be removed